t/45-contributions.t

```diff
@@ -31,7 +31,12 @@ subtest 'contribs' => sub {
 
 subtest 'multiple users' => sub {
     plan tests => 3;
-    my @contribs = $bot->contributions(['User:Mike.lifeguard', 'User:Reedy']);
+    my @contribs = $bot->contributions(['User:Mike.lifeguard', 'User:Reedy'], undef, undef,
+			'2015-01-01T00:00:00Z',
+			'2016-01-01T00:00:00Z',
+		);
+		use Data::Dumper;
+		print Dumper [0 + @contribs];
```

Comment:
Was this left over from debugging? If not, then either it should be outputted when the test fails (and not use `Data::Dumper` - use `diag` instead); or it should be logged instead of dumped directly to stdout.